md_core/mdpy/find_replace_bot/bot.py

```
@@ -0,0 +1,44 @@
+#!/usr/bin/python3
+"""
+
+python3 core8/pwb.py mdpy/find_replace_bot/bot
+
+"""
+import sys
+import os
+from pathlib import Path
```

Ruff

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)